rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +18,22 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise exceptions.ValidationError(msg)
+                    raise exceptions.ValidationError(
+                        msg,
+                        code='authorization'
```

Comment:
A lot of these strings might be common beyond DRF's internals. Reckon we could make them into constants? It might be easier for translation that way too.